Core/clim-core/gadgets.lisp

```diff
@@ -555,8 +555,9 @@ and must never be nil.")
 (defgeneric radio-box-selections (radio-box))
 
 (defmethod radio-box-selections ((pane radio-box))
-  (let ((v (radio-box-current-selection pane)))
-    (and v (list v))))
+  (loop for child in (sheet-children pane)
+        when (and (typep child 'toggle-button) (gadget-value child))
+        collect child))
```

Comment:
This seems unnecessary since the specification says:

> `radio-box [Class]`
> The instantiable class that implements an abstract radio box, that is, a gadget that constrains a number of toggle buttons, **only one of which may be selected at any one time**. It is a subclass of value-gadget and oriented-gadget-mixin.

Reading more, I'm not sure this method is about the currently selected choice at all. Compare

> `radio-box-current-selection radio-box [Generic Function]`
> Returns (or sets) the **current** selection for the radio box.

and

> `radio-box-selections radio-box [Generic Function]`
> Returns a sequence of **all** of the selections in the radio box.

I now think this should just return all children (maybe filtering out things that are not `toggle-button`s like you did).

Reply:
So returning all buttons regardless of gadget-value?

Reply:
Decided to push a commit doing this while I wait for an answer. I can always reset back to the previous commit if I'm wrong.

Reply:
afaik it's the master branch you want to avoid force pushing to

Reply:
> So returning all buttons regardless of gadget-value?

That's how I read it and @dkochmanski  agrees.

> afaik it's the master branch you want to avoid force pushing to

I guess it depends on the project but we absolutely rewrite history in pull request branches.